regression-test/suites/correctness_p0/test_null_predicate.groovy

```diff
@@ -115,4 +115,21 @@ suite("test_null_predicate") {
     """
 
     qt_select15 """ select * from test_null_predicate2 where `value` is null order by id; """
+
+    sql """ DROP TABLE IF EXISTS test_null_predicate"""
+    sql """
+        create table test_null_predicate (
+            id boolean null,
+            value int null
+        ) duplicate key(id)
+        DISTRIBUTED BY HASH(id) buckets 1
```

Comment:
set disable_auto_compaction to true. Or the data may be compacted and the case not stable.